### PR TITLE
Add translations where they were missing

### PR DESCRIPTION
**Pull request by mallikarai05**

### CODE REVIEW

File: src/translations/nl/translation.json [hunk 1]
```diff
@@ -88,5 +88,9 @@
   "Qtoogoodtobetrue": "Te goed om waar te zijn?",
   "Anstoogoodtobetrue": "Geloof ons niet, probeer het gewoon!",
   "ihavedifferentquestions": "Ik heb een andere vraag.",
-  "ihavenomorequestions": "Ik heb voorlopig geen vragen meer."
+  "ihavenomorequestions": "Ik heb voorlopig geen vragen meer.",
+  "search":"Zoekopdracht",
+  "privacy":"Privacy",
+  "sharedwithme": "Gedeeld met mij",
+  "collaborationinvites": "Samenwerking nodigt uit"
```
Comment (tijlleenders) [inline src/translations/nl/translation.json:95]:
```suggestion
  "collaborationinvites": "Samenwerkingsuitnodigingen"
```

File: src/translations/nl/translation.json [hunk 2]
```diff
@@ -88,5 +88,9 @@
   "Qtoogoodtobetrue": "Te goed om waar te zijn?",
   "Anstoogoodtobetrue": "Geloof ons niet, probeer het gewoon!",
   "ihavedifferentquestions": "Ik heb een andere vraag.",
-  "ihavenomorequestions": "Ik heb voorlopig geen vragen meer."
+  "ihavenomorequestions": "Ik heb voorlopig geen vragen meer.",
+  "search":"Zoekopdracht",
```
Comment (tijlleenders) [inline src/translations/nl/translation.json:92]:
```suggestion
  "search":"Zoek",
```

**Review by tijlleenders — APPROVED** [review]
Thanks & Happy new year! 🎇

**Review by tijlleenders — DISMISSED** [review]

**Comment by tijlleenders** [comment]
@Tushar-4781 Can we merge this?

**Comment by Tushar-4781** [comment]
Not right now. I c displayChangesModal file is interacted. It's still in use. Will merge after #868

**Comment by tijlleenders** [comment]
@mallikarai05 Can you resolve the conflicts?
This is ready to be merged.

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**